src/content/docs/en/tutorial/4-layouts/2.mdx

```diff
@@ -74,7 +74,7 @@ Here's an example of refactoring your code to include the `pubDate` in the layou
 ```markdown title="src/pages/posts/post-1.md" del={1}
 Published on: 2022-07-01
 
-Welcome to my new blog about learning Astro! Here, I will share my learning journey as I build a new website.
+Welcome to my _new blog_ about learning Astro! Here, I will share my learning journey as I build a new website.
```

Comment:
This line was added in Tutorial 2-2, and "my blog" should be italicized. 

https://github.com/withastro/docs/blob/8fbd915f1a6ac3da4d827490655fccc431ce22f8/src/content/docs/en/tutorial/2-pages/2.mdx#L58